app/react/portainer/settings/SettingsView/KubeSettingsPanel/KubeSettingsPanel.tsx

```diff
@@ -0,0 +1,109 @@
+import { Form, Formik } from 'formik';
+import { Settings as SettingsIcon } from 'lucide-react';
+import { useQueryClient } from 'react-query';
+
+import { notifySuccess } from '@/portainer/services/notifications';
+import { useEnvironmentId } from '@/react/hooks/useEnvironmentId';
+
+import { LoadingButton } from '@@/buttons';
+import { Widget } from '@@/Widget';
+
+import { useSettings, useUpdateSettingsMutation } from '../../queries';
+
+import { HelmSection } from './HelmSection';
+import { KubeConfigSection } from './KubeConfigSection';
+import { FormValues } from './types';
+import { DeploymentOptionsSection } from './DeploymentOptionsSection';
+import { validation } from './validation';
+
+export function KubeSettingsPanel() {
+  const settingsQuery = useSettings();
+  const queryClient = useQueryClient();
+  const environmentId = useEnvironmentId(false);
+  const mutation = useUpdateSettingsMutation();
+
+  if (!settingsQuery.data) {
+    return null;
+  }
+
+  const initialValues: FormValues = {
+    helmRepositoryUrl: settingsQuery.data.HelmRepositoryURL || '',
+    kubeconfigExpiry: settingsQuery.data.KubeconfigExpiry || '0',
+    globalDeploymentOptions: settingsQuery.data.GlobalDeploymentOptions || {
+      requireNoteOnApplications: false,
+      minApplicationNoteLength: 0,
+      hideAddWithForm: false,
+      hideFileUpload: false,
+      hideWebEditor: false,
+      perEnvOverride: false,
+    },
+  };
+
+  return (
+    <div className="row">
+      <div className="col-sm-12">
+        <Widget>
+          <Widget.Title icon={SettingsIcon} title="Kubernetes settings" />
```

Comment:
We need to use kube icon instead of setting icon. 

![image](https://github.com/portainer/portainer/assets/100548325/3c1038df-cebb-4196-80eb-0324387cee3a)